bentoml/_internal/bento/build_dev_bentoml_whl.py

```diff
@@ -60,7 +65,7 @@ def build_bentoml_editable_wheel(target_path: str) -> None:
                 ],
             )
             # copy the built wheel file to target directory
-            shutil.copytree(dist_dir, target_path)  # type: ignore
+            copytree(dist_dir, target_path, dirs_exist_ok=True)  # type: ignore
```

Comment:
could we change to use `shut.copy`?

Reply:
copy will throw error here since the directory is already created. What we can do is just to move the wheels file into the target path?